### PR TITLE
Added 8.3 Toys

### PR DESCRIPTION
**Pull request by criggs11**

### CODE REVIEW

File: app/data/toys.json [hunk 1]
```diff
@@ -39,6 +39,16 @@
             "icon": "inv_elemental_primal_water",
             "itemId": 168016,
             "name": "Hyper-Compressed Ocean"
+          },
+          {
+            "icon": "spell_nzinsanity_chasedbyshadows",
+            "itemId": 174830,
+            "name": "Shadowy Disguise"
```
Comment (seirl) [inline app/data/toys.json]: This is not specific to this expansion. Should go in the Pet Battles category, not BFA.

File: app/data/toys.json [hunk 2]
```diff
@@ -39,6 +39,16 @@
...
+          },
+          {
+            "icon": "inv_gnomish_xray_specs",
+            "itemId": 174871,
+            "name": "Mayhem Mind Melder"
```
Comment (seirl) [inline app/data/toys.json]: Same

File: app/data/toys.json [hunk 3]
```diff
@@ -127,6 +137,21 @@
             "icon": "inv_misc_trinket6oog_tablet1",
             "itemId": 170155,
             "name": "Carved Ankoan Charm"
+          },
+          {
+            "icon": "inv_knife_1h_grimbatolraid_d_03",
+            "itemId": 173951,
+            "name": "N'lyeth, Sliver of N'Zoth"
```
Comment (seirl) [inline app/data/toys.json]: Debatable but I'd rather have this in a BFA > World PvP subcategory

File: app/data/toys.json (hunk 1 above)
Comment (criggs11) [reply app/data/toys.json]: Fixed.

File: app/data/toys.json (hunk 2 above)
Comment (criggs11) [reply app/data/toys.json]: Fixed.

File: app/data/toys.json (hunk 3 above)
Comment (criggs11) [reply app/data/toys.json]: Fixed.

**Comment by seirl** [comment]
Thanks a lot for your work on the import!

**Pull request merged**